index.js

```diff
@@ -39,35 +40,23 @@ module.exports = function (content) {
     }
 
     var filePath = prefix + resource.slice(relativeToIndex + relativeTo.length); // get the base path
-    var html;
+    var html, match = content.match(/^module\.exports.+?(".*")/);
```

Comment:
> but it has a potential bug, that you don't consist on the quote type

If the content starts with a `"` and ends in `'` then it has a syntax error and is invalid.

 > And i think that's rare case some loader or webpack return single quoted string.

What do you base that on?